smartmin/tests.py

```diff
@@ -82,6 +85,11 @@ def setUp(self):
         self.user = None
         super(_CRUDLTest, self).setUp()
 
+    def run(self, result=None):
```

Comment:
It seems without nose, the test runner tries to run _CRUDLTest as a test... which this then prevents